bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/device/CommandHandler.java

```diff
@@ -384,15 +364,14 @@ public void handleCommand(InsteonChannelConfiguration conf, Command cmd, Insteon
         @Override
         public void handleCommand(InsteonChannelConfiguration conf, Command cmd, InsteonDevice dev) {
             try {
-                int button = this.getIntParameter("button", -1);
                 if (cmd == OnOffType.ON) {
-                    Msg m = dev.makeExtendedMessage((byte) 0x1f, (byte) 0x2e, (byte) 0x00,
-                            new byte[] { (byte) button, (byte) 0x09, (byte) 0x01 });
+                    Msg m = dev.makeExtendedMessage((byte) 0x1f, (byte) 0x20, (byte) 0x09, 
```

Comment:
Remove space at end of line, eclipse formatter removes it.

Reply:
Weird, not sure why me opening, saving in Eclipse didn't fix this. Anyway, Done